src/common/components/layouts/pages/inspect/Inspect.js

```diff
@@ -18,8 +18,14 @@ import PdfDocument from './pdfDocument/PdfDocument';
 import Structure from './structure/Structure';
 import DropzoneWrapper from '../upload/dropzoneWrapper/DropzoneWrapper';
 
+import errorTags from './validationErrorTags.json';
+
 import './Inspect.scss';
 
+export const GROUPS = _.map(_.entries(errorTags), category => category[0]);
+export const TAGS = _.flatMap(_.entries(errorTags), category => category[1]);
```

Comment:
This mapping can be replaced with a more readable one: `_.chain(a).values().flatten().value()`.